tests/Dfe.EarlyYearsQualification.UnitTests/Controllers/CheckAdditionalRequirementsControllerTests.cs

```diff
@@ -347,8 +448,7 @@ public async Task Post_PageContentIsReturned_MapsModelAndReturnsView()
 
         var answers = new Dictionary<string, string> { { "Test question", "yes" } };
```

Comment:
Question (very minor): `answers` is no longer used: is this right? If so, can be deleted 🙂

Reply:
Yep, good spot! Deleted now